helm-chart/banzai/templates/_helpers.tpl

```diff
@@ -84,8 +84,8 @@ External service URLS
   value: {{ .Values.banzai.configdbUrl | quote }}
 - name: OBSERVATION_PORTAL_URL
   value: {{ .Values.banzai.observationPortalUrl | quote }}
-- name: ES_URL
-  value: {{ .Values.banzai.elasticSearchURL | quote }}
+- name: OS_URL
```

Comment:
Can we stick to `OPENSEARCH_URL` for consistency across all projects?

Reply:
Yessir. Good call